.travis.yml

```diff
@@ -20,53 +20,75 @@ services: docker
 
 matrix:
   include:
-    - os: linux
-      env: MB_PYTHON_VERSION=2.7
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=2.7
-        - UNICODE_WIDTH=16
     - os: linux
       env:
-        - MB_PYTHON_VERSION=2.7
-        - PLAT=i686
+        - MB_PYTHON_VERSION=3.6
+        - BUILD_SDIST=true
     - os: linux
       env:
-        - MB_PYTHON_VERSION=2.7
+        - MB_PYTHON_VERSION=3.6
         - PLAT=i686
-        - UNICODE_WIDTH=16
     - os: linux
+      arch: arm64-graviton2
```

Comment:
are you sure you need to select this on travis side? I would think it is enough to tell multibuild which docker image to use (in your case "xenial_arm64v8"), and it should not matter which host linux you run docker from. Have you tried without setting arch, dist, virt, group etc and only exporting those multibuild environment variables?

Reply:
@anthrotype travis provides two types of instances for arm64 i.e. arm64 and arm64-graviton2, to use arm64-graviton2 we need to specify these settings like arch, dist, virt etc..